index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Cordoba Spring Festival -home</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&family=Lato:ital,wght@0,300;0,400;0,700;1,300;1,400;1,700&display=swap" 
+    rel="stylesheet">
+  <link rel="stylesheet" href="./styles/navbar.css">
+  <link rel="stylesheet" href="./styles/general.css">
+  <link rel="stylesheet" href="./styles/headline.css">
+  <link rel="stylesheet" href="./styles/program.css">
+  <link rel="stylesheet" href="./styles/musicians.css">
+  <link rel="stylesheet" href="./styles/footer.css">
+</head>
+<body>
+  <nav class="navBar">
+    <button class="hambugerMenu"  id="hamburgerMenu">
+      <img class="hambugerMenuImg" src="./assets/navbar/Icon - Menu.png" alt="menu">
+    </button>
+    <ul class="navbarList">
+      <li class="navbarListItem navbarFirst">
+        <a href="./index.html"><img class="orchestraIcon" src="./assets/navbar/orchestraIcon.jpg" alt="Cordoba Spring Festival 2023 Icon"></a> 
+      </li>
+      <li class="navbarListItem"><a class="navbarLink" href="./about.html">About</a></li>
+      <!--I believe this anchors should take the user to other pages, just like the about anchor above this comment does,
+      but for the purpose of this projects, does pages are not done yet. so the anchors are empty, to be compleated on a 
+      later date-->
+      <li class="navbarListItem"><a class="navbarLink">Program</a></li>
+      <li class="navbarListItem"><a class="navbarLink">Join</a></li>
+      <li class="navbarListItem"><a class="navbarLink">Sponsor</a></li>
+      <li class="navbarListItem"><a class="navbarLink">News</a></li>
+      <li class="navbarListItem lastItem"><a>Spring Festival</a></li>
+    </ul>
+  </nav>
+  <main>
+    <section class="headline">
+      <div class="headlineDiv">
+        <p class="miniTitle">"Hello! Music World"</p>
+        <h1 class="pageTitle">
+          CORDOBA CLASICAL<br/> SPRING FESTIVAL <br/>2023
+        </h1>
+        <p class="titleDescription">
+          Immerse yourself in the melodies of Beethoven, Mozart, 
+          and Handel at the Cordoba Spring Festival.
+          Experience the beauty of classical music in a captivating setting.
+        </p>
+        <h2 class="dateOfConcert">
+          2023.7.7(FRI) ~ 8(SAT)
+        </h2>
+        <p class="places">
+          @ Cordoba Real Theater, Art Center Güemes and more
+        </p>
+      </div>
+    </section>
+    <section class="program" id="sectionLink1">
+      <h3 class="programSectionTitle">
+        Main Program
+      </h3>
```

Comment:
- [ ] Nice work implementing the programs section :clap: However it would be nice to add the red line under the program title to follow the provided design(see image)
### Your design :point_down: 
![Screenshot from 2023-06-29 01-48-51](https://github.com/IsmaelMastronardi/Capstone1/assets/91142432/c0355e3d-7144-4d9b-bcae-84c86dbb1220)
### What it is supposed to look like :point_down: 
![Screenshot from 2023-06-29 01-49-15](https://github.com/IsmaelMastronardi/Capstone1/assets/91142432/3c7a1d99-f3f5-4920-8409-7f17f4f76c22)